src/components/educationColumn/EducationColumn.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import IconButton from "@mui/joy/IconButton";
+import DeleteOutlineIcon from "@mui/icons-material/DeleteOutline";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { AdapterDateFns } from "@mui/x-date-pickers/AdapterDateFns";
+import TextField from "components/UI/textField/TextField";
+import {
+  SDiv,
+  SWrapper,
+  deleteButtonSx,
+} from "components/educationColumn/style";
+import { IEducationColumnProps } from "components/educationColumn/typesDef";
+import DatePicker from "components/UI/datePicker/DatePicker";
+import i18next from "localization/en/en.json";
+
+function EducationColumn(props: IEducationColumnProps) {
+  const { type, deletable, id, removeHandler, index, control } = props;
+  const inputsProps =
+    type === "education"
+      ? {
+          helperTextPlace: i18next.ProfileEditForm.education.helperTextPlace,
+          helperTextPosition:
+            i18next.ProfileEditForm.education.helperTextPosition,
+          helperTextDateStart:
+            i18next.ProfileEditForm.education.helperTextDateStart,
+          helperTextDateEnd:
+            i18next.ProfileEditForm.education.helperTextDateEnd,
+          helperTextCity: i18next.ProfileEditForm.education.helperTextCity,
+          helperTextDescription:
+            i18next.ProfileEditForm.education.helperTextDescription,
+          formPlaceFieldName: "school",
+          formPositionFieldName: "degree",
+        }
+      : {
+          helperTextPlace: i18next.ProfileEditForm.experience.helperTextPlace,
+          helperTextPosition:
+            i18next.ProfileEditForm.experience.helperTextPosition,
+          helperTextDateStart:
+            i18next.ProfileEditForm.experience.helperTextDateStart,
+          helperTextDateEnd:
+            i18next.ProfileEditForm.experience.helperTextDateEnd,
+          helperTextCity: i18next.ProfileEditForm.experience.helperTextCity,
+          helperTextDescription:
+            i18next.ProfileEditForm.experience.helperTextDescription,
+          formPlaceFieldName: "employer",
+          formPositionFieldName: "jobTitle",
+        };
+
+  const handleRemoveButton = (currentId: string): void => {
+    removeHandler(currentId, index);
+  };
+
+  return (
+    <SWrapper>
+      {deletable ? (
```

Comment:
{deletable && (
        <IconButton
          sx={deleteButtonSx}
          onClick={() => handleRemoveButton(id)}
          variant="outlined"
        >
          <DeleteOutlineIcon />
        </IconButton>
      )} should work

Reply:
fixed